crates/ironrdp-glutin-renderer/src/surface.rs

```diff
@@ -238,12 +238,12 @@ impl Surface {
             let location = if let Some(location) = self.location.as_ref() {
                 location.clone()
             } else {
-                Rectangle {
-                    top: 0,
-                    left: 0,
-                    right: self.width - 1,
-                    bottom: self.height - 1,
-                }
+                Rectangle::new(
+                    0,
+                    0,
+                    self.width - 1,
+                    self.height - 1,
+                )
```

Comment:
I generally prefer struct initialization syntax when possible, that is when there is no private field and no code to run when creating the object. I prefer that for readability and flexibility:
- since all fields are named, there is no need to find the function definition in order to understand which value is what, and it’s easier to spot places where the wrong value is put into the wrong field (i.e.: putting height value into width)
- it’s possible to use the struct update syntax, etc

I understand you had to change the definition of `Rectangle` and that’s why this also had to change.

(Rest in the next comment)